docs/Tutorials/OpenAPI.md

```diff
@@ -780,11 +845,18 @@ As JSON, this could look as follows:
 
 ## OpenApi Documentation pages
 
-If you're not using a custom OpenAPI viewer, then you can use one of the inbuilt ones with Pode - either Swagger and ReDoc, or both!
+For each you can customise the Route path to access the page on, but by default Swagger is at `/swagger`, ReDoc is at `/redoc`, etc. If you've written your own custom OpenAPI definition then you can also set a custom Route path to fetch the definition on.
+
+If you're not using a custom OpenAPI viewer, then you can use one or more of the inbuilt which Pode supports: ones with Pode:
 
-For both you can customise the path to access the page on, but by default Swagger is at `/swagger` and ReDoc is at `/redoc`. If you've written your own custom OpenAPI definition then you can also set a custom path to fetch the definition.
+* Swagger
+* ReDoc
+* RapiDoc
+* StopLight
+* Explorer
+* RapiPdf
```

Comment:
These two sections appear to have been swapped around, which throws the wording off. The paragraph at line 848 should come after the bullet-point list of OpenAPI viewers.